src/ReleaseApp/Domain/Entities/Collection/UpgradeInstructionsReleaseGroupCollection.php

```diff
@@ -88,6 +88,30 @@ public function sortByReleasedDate(): self
         return new self(array_values($sortData));
     }
 
+    /**
+     * @return self
+     */
+    public function getOnlySecurityFixes(): self
+    {
+        return new self(
+            array_filter($this->elements, function ($releaseGroup) {
```

Comment:
static fn(...): bool => $releaseGroup->isSecurity();

Add type-hints to the function

Reply:
++